### PR TITLE
Add content for machine learning in the index/welcome

### PR DESCRIPTION
**Pull request by theckang**

### CODE REVIEW

File: workshop/content/index.md [hunk 1]
```diff
@@ -1,20 +1,21 @@
-# OpenShift Serverless
+# OpenShift Serverless with Machine Learning
 
 ## Description 
 
-Serverless computing allows developers to optimize their application usage by executing and billing on application resource consumption.  Traditionally, this execution model was offered by cloud providers in a non-portable manner.  Today, the serverless paradigm is offered on OpenShift - leveraging the innovation happening in the open source Knative project. This brings a key capability for developers to build portable cloud native applications in hybrid and multi-cloud environments.
+Serverless computing allows developers to optimize their application usage by executing and billing on application resource consumption.  Traditionally, this execution model was offered by cloud providers in a non-portable manner.  Today, the serverless paradigm is offered on OpenShift - leveraging the innovation happening in the open source Knative project. This brings a key capability for developers to build portable cloud native applications in hybrid and multi-cloud environments.
```
Comment (jkeam) [inline workshop/content/index.md]: Nitpick: Delete trailing whitespace.

**Comment by jkeam** [comment]
Question: Since the ML part is part 2 of the workshop, I've always wondered where the introduction for it should go.  If you think about the presenter/admin/person giving the workshop, should the ML part be part of this here?

**Comment by theckang** [comment]
> Question: Since the ML part is part 2 of the workshop, I've always wondered where the introduction for it should go. If you think about the presenter/admin/person giving the workshop, should the ML part be part of this here?

I definitely think we need to mention the ML part in the "Index" which is an overview of everything that will be covered.  In terms of specifics, Lab 1 (your stuff) intros serverless and Lab 2( my stuff) intros the ML part.

**Review by jkeam — APPROVED** [review]
looks good

**Pull request merged**